features/mbedtls/targets/TARGET_CRYPTOCELL310/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Guidelines for porting Cryptocell to Mbed OS
```

Comment:
That should go to the Handbook repo.

Reply:
I don't think so. This is not a generic porting guide. Quoting @AnotherButler about this (through mail):
>. We don’t include board- or module-specific information in the Handbook. However, if there’s porting information that’s applicable to all boards or modules, we’ll happily put that in the Handbook. If it’s board-specific, let’s keep it in the README. 

This is not board specific but it's component specific